src/Workspaces/Remote/ServiceHub.CoreComponents/Microsoft.CodeAnalysis.Remote.ServiceHub.CoreComponents.csproj

```diff
@@ -18,18 +18,21 @@
 
   <Target Name="PublishProjectOutputGroup" DependsOnTargets="Publish" Returns="@(_PublishedFiles)">
     <ItemGroup>
-      <!-- Need to include and then update items (https://github.com/microsoft/msbuild/issues/1053) -->
-      <_PublishedFiles Include="$(PublishDir)**\*.*" Exclude="$(PublishDir)**\Microsoft.CodeAnalysis.Remote.ServiceHub.CoreComponents.*"/>
-      <_PublishedFiles Remove="@(_PublishedFiles)" Condition="'%(Extension)' == '.pdb'" />
-
+      <_ExcludedFiles Include="$(PublishDir)**\Microsoft.CodeAnalysis.Remote.ServiceHub.CoreComponents.*" />
+      <_ExcludedFiles Include="$(PublishDir)**\*.pdb" />
+      <!-- the only assembly we need under runtime folder (runtimes\win-x64\native\e_sqlite3.dll) is handled by the vsix project directly -->
+      <_ExcludedFiles Include="$(PublishDir)\runtimes\**\*.*" />
```

Comment:
Is this going to cause any surprises down the road if somebody is trying to add something else and this breaks it?

Reply:
I agree this is fragile (so does the way we pick and drop the sqlite dll in the roslyn vsix project.) I'm hoping that our integration test would catch it with newly added debug.assert